docs/extensions/bridge/bridge.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Bridge
+---
+
+import {
+  DiscordButton,
+  DiscordButtons,
+  DiscordInteraction,
+  DiscordMessage,
+  DiscordMessages,
+} from "@discord-message-components/react";
+import "@discord-message-components/react/styles";
+
+## Concept
+
+Let's say that you want to make a command that is both a Slash Command and a Prefixed Command. Now, you could just copy and paste the code from the first command callback to the other and make some adjustments, but that's not very efficient.
+
+This is where the `ext.bridge` comes in. It allows you to use one callback to make both a Slash Command and a Prefixed Command.
+
+:::note
+Currently, `ext.bridge` has not been merged to the main branch yet and it's somewhat experimental. So in order to use it, you'll have to install the `py-cord` package from the `ext-compat` branch by running this:
+```
+pip install git+https://github.com/Pycord-Development/pycord.git@ext-compat
+```
+:::
```

Comment:
I think we'll have it merged into the library by the time we merge this into the guide.
```suggestion
```